src/nanoemoji/nanoemoji.py

```diff
@@ -489,6 +475,56 @@ def _generate_color_font(config: ColorFontConfig, inputs: Iterable[InputGlyph]):
     return ufo, ttfont
 
 
+def self_dir(path):
+    return os.path.dirname(os.path.abspath(__file__))
+
+
+def build_dir():
+    return os.path.abspath(FLAGS.build_dir)
+
+
+def rel_self(path):
+    self_dir = self_dir()
+    path = os.path.normpath(os.path.join(self_dir, path))
+    return os.path.relpath(path, self_dir)
+
+
+def rel_build(path):
+    return os.path.relpath(path, build_dir())
+
+
+def resolve_rel_build(path):
+    return os.path.abspath(os.path.join(build_dir(), path))
+
+
+def write_preamble(nw):
+    nw.comment('Generated by nanoemoji')
+    nw.newline()
+
+    nw.rule('picosvg', 'picosvg $in > $out || echo "$in failed picosvg"')
+    nw.rule('codepointmap', 'nanoemoji_write_codepoints $in > $out')
```

Comment:
how about instead of installing setuptools CLI scripts for write_codepoints and write_fea (just to make the executable by ninja rules), we invoke the modules as scripts with `python -m nanoemoji.write_codepoints`? That basically has the effect of importing a module with its name set to "__main__" (which will then trigger the main() entry point).

Actually, we want `f"{sys.executable} -m nanoemoji.write_codepoints"` rather than unqualified `python`, so that we will reuse the same exact interpreter that was used to run nanoemoji in the first place.

Reply:
Oh good idea, will update. I don't think I've seen sys.executable before, that's cool

Reply:
Done